### PR TITLE
Add multisample shortcut and parameters

### PR DESCRIPTION
**Pull request by dev7355608**
##### Description of change

- Added getter and setter shortcut for `renderTexture.framebuffer.multisample`.
- Added multisample option to `RenderTexture.create()` and `RenderTexture`/`BaseRenderTexture` constructors.
- Added multisample parameter to `AbtractRenderer.generateTexture()`.

##### Pre-Merge Checklist

- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: packages/core/src/renderTexture/BaseRenderTexture.ts [hunk 1]
```diff
@@ -70,14 +72,15 @@ export class BaseRenderTexture extends BaseTexture
             const height = arguments[1];
             const scaleMode = arguments[2];
             const resolution = arguments[3];
+            const multisample = arguments[4];
```
Comment (bigtimebuddy) [inline packages/core/src/renderTexture/BaseRenderTexture.ts:75]: This isn't necessary, backward compatibility not needed in this case)

File: packages/core/src/renderTexture/RenderTexture.ts [hunk 2]
```diff
@@ -151,8 +167,10 @@ export class RenderTexture extends Texture
      * @param {number} [height]
      * @param {PIXI.SCALE_MODES} [scaleMode=PIXI.settings.SCALE_MODE]
      * @param {number} [resolution=1]
+     * @param {PIXI.MSAA_QUALITY} [multisample=PIXI.MSAA_QUALITY.NONE]
      */
-    static create(width: number, height: number, scaleMode?: SCALE_MODES, resolution?: number): RenderTexture;
+    static create(width: number, height: number, scaleMode?: SCALE_MODES, resolution?: number,
+        multisample?: MSAA_QUALITY): RenderTexture;
```
Comment (bigtimebuddy) [inline packages/core/src/renderTexture/RenderTexture.ts:173]: Same, please don't add more to deprecated methods. The approach here is object-based param.

File: packages/core/src/renderTexture/RenderTexture.ts [hunk 3]
```diff
@@ -181,6 +200,7 @@ export class RenderTexture extends Texture
                 height: rest[0],
                 scaleMode: rest[1],
                 resolution: rest[2],
+                multisample: rest[3],
```
Comment (bigtimebuddy) [inline packages/core/src/renderTexture/RenderTexture.ts:203]: Unnecessary.

**Review by bigtimebuddy — CHANGES_REQUESTED** [review]
Don't have any problem overall with adding some of these new options. Just some comments where we don't need to supported this on deprecated implementations.

**Review by ivanpopelyshev — APPROVED** [review]
Looks good!

**Review by bigtimebuddy — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
@dev7355608 could you resolve conflicts with your other PR?

**Comment by bigtimebuddy** [comment]
Thank you @dev7355608 super useful shortcuts APIs here.

**Pull request merged**